src/popup/App.vue

```diff
@@ -118,12 +118,8 @@
         </div>
       </div>
       <div class="button__container">
-        <md-button v-if="checkedLogs.length" class="md-raised md-accent" @click="syncToJira">
-          <span v-show="!isSaving">Log work</span>
-          <span v-show="isSaving">Logging...</span>
-        </md-button>
-        <md-button v-else disabled class="md-raised md-accent" @click="syncToJira">
-          <span>Log work</span>
+        <md-button :disabled="isRefreshingEntries || isSyncing" class="md-raised md-accent" @click="syncToJira">
```

Comment:
Should this also keep the check that there is at least one checked log? "checkedLogs.length"